docs/pages/User/Configure/network.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Configure Liqo
+weight: 2
+---
+
+## Gateway
+
+You have to label one node of your cluster as the gateway node. This will be used as the gateway for the inter-cluster traffic.
```

Comment:
All the inter-cluster traffic is delivered through a special node that acts as _gateway_ between the local cluster and the other peered clusters.
This node has to be manually labelled as `gateway`.